apps/registry/pages/api/[payload].js

```diff
@@ -172,6 +172,7 @@ ${JSON.stringify(validation.errors, null, 2)}
   try {
     formatted = await formatter.format(selectedResume, options);
   } catch (e) {
+    console.error(e); // Log the error on the server so we see it in the logs
```

Comment:
I had some issues with properly calling the functions from the package `jsonresume-theme-standard` and it was a little tricky to debug without having the errors logged to the server console. I think we should not only send the error messages to the users but also have them on the server.